### ISSUE TITLE
For non-resizeable attachments, like PDFs, displays "Resize disabled: unsupported file format"

### ISSUE THREAD

**Issue by Betterbird** [memeller/shrunked#13]
Please suppress this menu entry, it's confusing, no one wants to resize PDFs.

![image](https://github.com/memeller/shrunked/assets/86832723/b4f1218c-8146-4932-85ef-b63f4caa763e)

**Comment by GoczkuPL** [comment]
Why no one? Compressing PDFs will be something fabulous for some users :)

**Comment by Betterbird** [comment]
Well, whichever type can't be compressed shouldn't show the menu entry.

**Comment by memeller** [comment]
This was added in one of the first commits when, for example, bmp files were not supported. I added this information because I was confused as to why there was no option to resize the bmp file when right clicking on it, because I had forgotten that bmp files were not supported.
While showing this information (rather than hiding the menu item) still seems like a better solution for me, I also accept that other users may not share this view, so I will add the option to change the behaviour of the context menu.
